contracts/src/v0.8/ccip/rmn/RMNRemote.sol

```diff
@@ -45,11 +45,10 @@ contract RMNRemote is OwnerIsCreator, ITypeAndVersion, IRMNRemote {
   }
 
   /// @dev the contract config
-  /// @dev note: minSigners can be set to 0 to disable verification for chains without RMN support
   struct Config {
     bytes32 rmnHomeContractConfigDigest; // Digest of the RMNHome contract config
-    Signer[] signers; // List of signers
-    uint64 minSigners; // Threshold for the number of signers required to verify a report
+    Signer[] signers; //                    List of signers
+    uint64 f; //                            Max number of faulty RMN nodes; f+1 signers are required to verify a report
```

Comment:
Says f+1 signers are required to verify but below we assert `newConfig.signers.length < 2 * newConfig.f + 1)`. Can you clarify this with some more comments as it's confusing to me